{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -3,9 +3,13 @@
 """
 
 import environ
+import os
 
 ROOT_DIR = environ.Path(__file__) - 3  # ({{ cookiecutter.project_slug }}/config/settings/base.py - 3 = {{ cookiecutter.project_slug }}/)
 APPS_DIR = ROOT_DIR.path('{{ cookiecutter.project_slug }}')
+{% if cookiecutter.js_task_runner == 'CreateReactApp' -%}
```

Comment:
Consistency is key -- make sure to rename both `CreateReactApp` and `createreactapp` to `create_react_app` everywhere, and use that spelling universally.